src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -766,9 +763,10 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
           'aria-expanded': isCardExpanded,
         }}
       >
+        <CardActions>{...props.actions || []}</CardActions>
         <CardTitle component="h4">
-          <Split>
-            <SplitItem isFilled>Automated Analysis</SplitItem>
+          <Split hasGutter>
```

Comment:
Maybe we should move the label out of the card title to avoid title-styles applied to the label.

It would look like this. Seems to align better and follow Patternfly guide.

![Screenshot from 2022-12-08 18-35-49](https://user-images.githubusercontent.com/68053619/206589240-36fdb6e0-58b3-4535-8a8e-3daa81c6b12c.png)



```diff
diff --git a/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx b/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
index 98e69b4..275b308 100644
--- a/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
+++ b/src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx
@@ -78,6 +78,7 @@ import {
   GridItem,
   Label,
   LabelGroup,
+  Level,
   Split,
   SplitItem,
   Stack,
@@ -764,12 +765,10 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
         }}
       >
         <CardActions>{...props.actions || []}</CardActions>
-        <CardTitle component="h4">
-          <Split hasGutter>
-            <SplitItem>Automated Analysis</SplitItem>
-            <SplitItem>{reportSource}</SplitItem>
-          </Split>
-        </CardTitle>
+        <Level hasGutter>
+          <CardTitle component="h4">Automated Analysis</CardTitle>
+          {reportSource}
+        </Level>
       </CardHeader>
       <CardExpandableContent>
         <Stack hasGutter>
```